test/distributions/test_constraints.py

```diff
@@ -7,6 +7,13 @@
 from torch.testing._internal.common_cuda import TEST_CUDA
 
 
+EXAMPLES = [
+    (constraints.symmetric, False, [[2., 0], [2., 2]]),
+    (constraints.positive_definite, False, [[2., 0], [2., 2]]),
+    (constraints.symmetric, True, [[3., -5], [-5., 3]]),
+    (constraints.positive_definite, False, [[3., -5], [-5., 3]]),
```

Comment:
Can we add a test case with a (B, k, k) shaped tensor where the positive definite check returns True? I know that this should be covered by a MultivariateNormal normal test but it will be good to test this explicitly with a batched input case.

Reply:
Sure, it is my pleasure.